pkg/harvester/config/table-headers.js

```diff
@@ -42,3 +42,19 @@ export const SNAPSHOT_TARGET_VOLUME = {
   sort:      'spec.source.persistentVolumeClaimName',
   formatter: 'SnapshotTargetVolume',
 };
+
+// The column of total snapshot quota column on namespace list page
+export const NS_SNAPSHOT_QUOTA = {
```

Comment:
Could we remove it and use instead the one defined in the shell?

Reply:
Sure, they're duplicated.